src/ci/bin/test.inc.sh

```diff
@@ -312,14 +311,11 @@ cromwell::private::pull_common_docker_images() {
     docker pull ubuntu
 }
 
-cromwell::private::install_cwltool() {
+cromwell::private::install_cwltest() {
     # TODO: No clue why these are needed for cwltool. If you know please update this comment.
```

Comment:
is this the time to update this comment?

Reply:
I still don't know why this is required. 🤔 So I'm leaving it until someone gives an answer.